pkg/virt-api/webhooks/validating-webhook/admitters/vmrestore-admitter.go

```diff
@@ -203,10 +203,19 @@ func (admitter *VMRestoreAdmitter) validateCreateVM(field *k8sfield.Path, namesp
 	}
 
 	if rs != v1.RunStrategyHalted {
-		cause := metav1.StatusCause{
-			Type:    metav1.CauseTypeFieldValueInvalid,
-			Message: fmt.Sprintf("VirtualMachine %q is running", name),
-			Field:   field.String(),
+		var cause metav1.StatusCause
+		if vm.Spec.Running != nil && *vm.Spec.Running {
+			cause = metav1.StatusCause{
+				Type:    metav1.CauseTypeFieldValueInvalid,
+				Message: fmt.Sprintf("VirtualMachine %q is running", name),
```

Comment:
It may make sense to write `VirtualMachine %q is not halted`, since technically the VM may indeed not be running. Just the desire to have it running is expressed.

Reply:
done